src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -429,8 +440,8 @@ def parse_rpc_arg(rpc_arg: str) -> Sequence[str]:
         type=int,
         help="The number of channels from which to send requests.")
     parser.add_argument("--print_response",
-                        default=False,
-                        action="store_true",
+                        default="False",
+                        type=str,
```

Comment:
Why change this from a bool to str? Or that is the convention of other clients/servers?

Reply:
This is how the other clients are doing it. I find it a little weird, but it has to be done to work.

Reply:
This is the convention of the other clients. I find it a little weird, but I think consistency is the most important thing.